src/components/LifecycleItem.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const LifecycleItem = (props) => {
+  return (
+    <div className='flex relative bg-white flex-col justify-center items-center px-4 py-8 rounded-3xl text-base'>
+      <img className='max-w-none absolute -top-12 hidden md:block' src={props.data.icon} alt={props.data.title} />
+      <img className='max-w-none absolute -top-2 md:hidden block' src={props.data.iconMobile} alt={props.data.title} />
```

Comment:
Issue: The `alt` text for these images isn't descriptive. If the images are purely decorative you can use `alt=""` . Otherwise, I'd suggest adding alt text to the `lifecycleData` objects.